packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -224,7 +233,7 @@ async function registerTestView(
   // Refresh Test View command
   testViewItems.push(
     vscode.commands.registerCommand('sfdx.force.test.view.refresh', () => {
-      if (!LanguageClientUtils.indexing) {
+      if (!languageClientUtils.getStatus().isIndexing()) {
```

Comment:
@brpowell I was wondering if we should change this from when it's not indexing to when it's ready. What do you think ?

Reply:
We could combine ready and indexing as well. I think if the language server is ready it should mean it started successfully and completed indexing.